### ISSUE TITLE
Line flickering and disappearing on Chrome 43 (Windows and MacOS)

### ISSUE THREAD

**Issue by mh-nichts** [wentin/underlineJS#30]
First of all, thanks for this nice experiment!

I've been playing with it, wanting to use it for a future project, but I realize now that the animation doesn't look nice on Chrome : the canvas line is blinking/flickering, and even disappearing sometimes.
I can see it on the demo page as well.

Everything work as expected on Firefox 39 and IE11 (Windows), and on Firefox 37 and Safari 6.1 (MacOS).

The thing is, I'm almost sure that it was working properly some months ago (in may), so maybe this is linked to a chrome update.
Have you had any feedback or discoveries on that point ?

**Comment by wentin** [comment]
yes, i am bummed about it too! it was working before with chrome as it was my dev environment. I will dig into what is going on with the latest chrome that interfere with underline. I am suspecting, refreshing frame rate doesn't match?

**Comment by mh-nichts** [comment]
That's also the direction I started looking in, but I couldn't find anything about the last Chrome updates causing this problem.
Nevertheless, I found this bug, who looks similar, but it goes back to Chrome 38 (and I don't understand everything that is discussed) : https://code.google.com/p/chromium/issues/detail?id=422000
Also I've seen you used requestAnimationFrame, but I'm really not familiar with it so I can't even think of what to try and test...

**Comment by mh-nichts** [comment]
Hello,
Just so you know, it seems it has been solved with the last update (v44) !

**Comment by wentin** [comment]
what, realy! woohoo, i am get test it

thanks for the good news!

On Mon, Jul 27, 2015 at 9:37 AM, mh-nichts <notifications@github.com> wrote:

> Hello,
> Just so you know, it seems it has been solved with the last update (v44) !
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wentin/underlineJS/issues/30#issuecomment-125206656>.
>



-- 
Best regards,
Wenting
Wenting's Portfolio
<http://t.sidekickopen13.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJN7t5XZscW858W8rBN947gsh1-VQBb2Y56dGmKf3Wr37l02?t=http%3A%2F%2Fwentin.co%2F&si=5007677723770880&pi=d3b35cac-d76e-4646-84e4-f5a5f35d1bb3>
